lib/connection_manager.go

```diff
@@ -156,15 +162,19 @@ func NewConnectionManager(
 		//newestBlock: _newestBlock,
 
 		// Initialize the peer data structures.
-		outboundConnIPGroups:   make(map[string]int),
-		persistentPeers:        make(map[uint64]*Peer),
-		outboundPeers:          make(map[uint64]*Peer),
-		inboundPeers:           make(map[uint64]*Peer),
-		connectedOutboundAddrs: make(map[string]bool),
+		outboundConnIPGroups:       make(map[string]int),
```

Comment:
It's worth adding comments here to explain what the key is for each of these maps. I'm assuming it's PeerID -> Peer, and PeerID -> OutboundConnectionAttempt,... A comment will clarify for future readers

Reply:
We got a comment on this in the ConnectionManager struct definition. Should we make it more verbose?

Reply:
For outboundConnectionAttempts it's actually attemptId -> OutboundConnectionAttempt. I added a comment to clarify this.